main.go

```diff
@@ -82,6 +73,26 @@ func main() {
 	}
 }
 
+// readConfig to read config and generate tasks
+func readConfig(config string) ([]*walker.Task, error) {
+	b, err := ioutil.ReadFile(config)
+	if err != nil {
+		return nil, fmt.Errorf("Can't find config file %v", err)
+	}
+
+	if len(files) == 0 {
```

Comment:
Maybe the "len(files)" validation shouldn't be in this function?

Reply:
We still need the condition there for me to test that part of code but I forgot to pass file variable as a function parameter.